client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/purchase-section/index.js

```diff
@@ -39,6 +40,16 @@ const PurchaseSection = props => {
 	} = props;
 	const purchaseBusy = form.isSubmitting && ! form.needsPrintConfirmation;
 	const hasSelectedRate = hasSelectedRates( form.rates );
+	let labelRequiresPaymentMethod = false;
+	if( hasSelectedRate ) {
+		forEach( form.rates.values , ( label ) => {
```

Comment:
hey @bborman22 , what do you think about replacing this `forEach` with its ES6 version, `for..of`? That way we can reduce to the minimum the amount of lodash imports. I've been trying to do it lately as much as possible (reduce lodash dependencies, I mean).

Reply:
Yeah I agree, I'll look to convert this to ES6 to remove the dependency.